Delphi.WebMock.RequestStub.pas

```diff
@@ -46,9 +46,18 @@   TWebMockRequestStub = class(TObject)
       : TWebMockResponse;
     function WithBody(const AContent: string): TWebMockRequestStub; overload;
     function WithBody(const APattern: TRegEx): TWebMockRequestStub; overload;
+    /// <summary>
+    ///   Works with TMultipartFormData Fields
+    /// </summary>
+    /// <example>
+    ///   <code lang="Delphi">// FogBugz submission
+    /// WebMock.StubRequest('POST', '/api')
+    ///     .WithBody('cmd', 'new')
+    ///     .ToRespond.WithBody('&lt;?xml version="1.0" encoding="UTF-8"?&gt;&lt;response&gt;&lt;case ixBug="459" operations="edit,spam,assign,resolve"&gt;&lt;/case&gt;&lt;/response&gt;');</code>
+    /// </example>
+    function WithBody(const ADataName, ADataValue: string): TWebMockRequestStub; overload;
```

Comment:
I feel like this should be named `WithFormData` as it is quite specifically for that. Then the arguments would be shortened to `AName` and `AValue` making it read better overall. On the other hand, it also replaces the body matcher so might be confusing behaviour if someone was to use `WithBody('Some Text').WithFormData('User', 'Bob')` without making the connection that they are the _same_ thing. Do you have any thoughts on this?

Reply:
First time in I simply use what pops up in the list and either one of those would have worked for me. 
However there are more kinds of form data so mull over the trailing Enum parameter I proposed above.